chore: dfx copy updates (#586)
dfinity/docs: modules/developers-guide/pages/cli-reference/dfx-ping.adoc

= dfx ping

> Use the `+dfx ping+` command to check connectivity between a local computer and an {IC} network provider.

Reviewer: ```suggestion
Use the `+dfx ping+` command to check connectivity between a local computer and the {IC} blockchain.
```

> This command enables you to verify that you can connect to a specific {IC} network and that the remote network services are running. By default it will ping the {platform}, but can also be used to ping test networks.

Reviewer: ```suggestion
This command enables you to verify that you can connect to the {IC} blockchain.
```

> This command enables you to verify that you can connect to the environment where you want to deploy to. By default it will ping the {platform}, but can also be used to ping test networks.

Reviewer: ```suggestion
This command enables you to verify that you can connect to the environment where you want to deploy to. 
```

Note that you can only run this command from within the project directory structure.

> For example, if your project name is `+hello_world+`, your current working directory must be the `+hello_world+` top-level project directory or one of its subdirectories.

Reviewer: L37: omit "network provider"

|===
|Argument |Description

> |provider |Specifies the testnet URL that you want to use.

Reviewer: ```suggestion
|provider |Specifies the {platform} or testnet URL that you want to use.
```
